### PR TITLE
Remove va opt

### PR DESCRIPTION
**Pull request by felipebalbi**
Depends on #9

### CODE REVIEW

File: lib/flux.h [hunk 1]
```diff
@@ -123,7 +123,7 @@ extern ValueHeader *flux_script_eval_string(char *script_string);
 
 #define PANIC(message, ...)                                        \
   flux_log("\n\e[1;31mPANIC\e[0m in \e[0;93m%s\e[0m: ", __func__); \
-  flux_log(message __VA_OPT__(,) __VA_ARGS__);                     \
+  flux_log(message, __VA_ARGS__);                                  \
```
Comment (daviwil) [inline lib/flux.h]: Interesting, I thought `__VA_OPT__` was necessary if the macro can also be called without any arguments passed.

Comment (felipebalbi) [reply lib/flux.h]: Only for C++, but I just noticed there's a mistake in my PR which I'll correct shortly, we need the paste character `#` before `__VA_ARGS__` for it to work. Here's a quick test:

```c
#define test0(fmt, ...) printf(fmt, __VA_ARGS__)
#define test1(fmt, ...) printf(fmt, ##__VA_ARGS__)

int main(void)
{
	test0("hello %s", "world");
	test0("hi");

	test1("hello %s", "world");
	test1("hi");

	return 0;
}
```

Running through the preprocessor we get:

```c
$ gcc -E test.c 
# 0 "test.c"
# 0 "<built-in>"
# 0 "<command-line>"
# 1 "/usr/include/stdc-predef.h" 1 3 4
# 0 "<command-line>" 2
# 1 "test.c"



int main(void)
{
 printf("hello %s", "world");
 printf("hi", );

 printf("hello %s", "world");
 printf("hi");

 return 0;
}
```

Comment (daviwil) [reply lib/flux.h]: Interesting, I didn't know about that!  Thanks a lot for pointing it out

**Comment by daviwil** [comment]
Next I'm going to fix this CI issue so that we can see legit build/test results on PRs.

**Comment by felipebalbi** [comment]
> Next I'm going to fix this CI issue so that we can see legit build/test results on PRs.

I think the CI issue is because ubuntu fails to find whatever package is needed as a dependency. Perhaps add `sudo apt-get update` before installing deps?

**Comment by daviwil** [comment]
Yep, that's what I figured the problem was.  I'm also going to merge `scripting` into `master` since we're more than likely going to continue on the scripting language path for now.

**Pull request merged**